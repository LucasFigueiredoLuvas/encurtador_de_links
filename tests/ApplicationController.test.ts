import { TestServer } from "./jest.setup";

describe("ApplicationController requests and responses", () => {

  it("root path should return status-code 200", async () => {
    const response = await TestServer.get("/");
    expect(response.status).toBe(200);
    expect(response.text).toMatchSnapshot();
  });

});
