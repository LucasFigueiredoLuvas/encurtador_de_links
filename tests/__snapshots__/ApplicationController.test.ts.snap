// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationController requests and responses Should return status-code 200 1`] = `
"<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Tin URL ny</title>
</head>
<body>
  <span>url</span>
</body>
</html>"
`;
